### PR TITLE
docs: Added the Python version Badge and Logos to badges fixes: #1052

### LINKED ISSUE #1052: Add a badge for Python version in README.md
## Description
README.md should have a badge for denoting the version of python needed.

## Mocks
README of [mentorship-backend](https://github.com/anitab-org/mentorship-backend/blob/develop/README.md).

Screenshot from 2020-08-29 20-56-52

## Definition of Done
- [ ] Badge should be added with appropriate version.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by atharwa-24**
# Description
Added the Python version badge to the README file. Additionally added logos to the Travis ,Zulip and Python Badge in README file.

Fixes #1052 

# Type of Change:
doc:

- Documentation

# How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.
## before
vms before badge

## after
vms after badge

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
 AnitaB.org Portal - VMS project
 ============================
 
-[![Build Status](https://travis-ci.org/anitab-org/vms.svg?branch=develop)](https://travis-ci.org/anitab-org/vms) [![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg?branch=develop)](https://coveralls.io/github/anitab-org/vms?branch=develop) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)
+
```
Comment (rkpattnaik780) [inline README.md:4]: Remove this blank line.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+[![Build Status](https://img.shields.io/travis/anitab-org/vms/develop?logo=travis)](https://travis-ci.org/anitab-org/vms)
+[![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg)](https://coveralls.io/github/anitab-org/vms?branch=develop) 
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com) 
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg?logo=zulip)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)
+[![Python Version](https://img.shields.io/badge/python-3.6-blue.svg?logo=python)](https://www.python.org/downloads/release/python-360/)
+
```
Comment (rkpattnaik780) [inline README.md:10]: Remove this blank line.

**Review by rkpattnaik780 — CHANGES_REQUESTED** [review]
Functionality has been added. Just a minor change is required for approval.

**Comment by atharwa-24** [comment]
Hey thanks for this one i wasn't known to the concept of Updating pull requests 
learnt to apply the changes in the existing PR

**Review by SanketDG — APPROVED** [review]
Looks good to me, if you can squash the commits I will merge this!

**Comment by atharwa-24** [comment]
ok @SanketDG

**Comment by atharwa-24** [comment]
@SanketDG Done

**Comment by atharwa-24** [comment]
@SanketDG @rkpattnaik780 Please review the changes

File: README.md [hunk 3]
```diff
@@ -1,8 +1,11 @@
 AnitaB.org Portal - VMS project
 ============================
-
-[![Build Status](https://travis-ci.org/anitab-org/vms.svg?branch=develop)](https://travis-ci.org/anitab-org/vms) [![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg?branch=develop)](https://coveralls.io/github/anitab-org/vms?branch=develop) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)
-
+[![Build Status](https://img.shields.io/travis/anitab-org/vms/develop?logo=travis)](https://travis-ci.org/anitab-org/vms)
```
Comment (rkpattnaik780) [inline README.md:3]: Keep one new line after the second line. Earlier commits had two lines hence I asked to remove one.

**Comment by rkpattnaik780** [comment]
Add “fixes #” in the commit body as well.

**Comment by rkpattnaik780** [comment]
The commits needed to be squashed, any reason you closed this PR @atharwa-24

**Comment by atharwa-24** [comment]
i had messed up while squashing the PR @rkpattnaik780

**Comment by atharwa-24** [comment]
I will make the changes and will put a new PR

**Comment by rkpattnaik780** [comment]
Try to fix this PR. Let me know if you need help.

**Comment by atharwa-24** [comment]
> Try to fix this PR. Let me know if you need help.

I tried to fix that one but its not reopening

**Comment by atharwa-24** [comment]
Can we just select certain commits and make the PR

**Comment by rkpattnaik780** [comment]
> Can we just select certain commits and make the PR

Okay, make a fresh PR then.

**Pull request closed without merging**